pkg/manifests/types.go

```diff
@@ -343,6 +343,8 @@ type NodeExporterCollectorNetDevConfig struct {
 type NodeExporterCollectorNetClassConfig struct {
 	// A Boolean flag that enables or disables the `netclass` collector.
 	Enabled bool `json:"enabled,omitempty"`
+	// A Boolean flag that activate the netlink implementation of the `netclass` collector.
```

Comment:
Anything worth mentioning for the users like different metric names?

Reply:
yup, it will lose 3 metrics `node_network_address_assign_type`, `node_network_name_assign_type`, `node_network_device_id`.
I will update the comment.

Reply:
```suggestion
	// A Boolean flag that activates the netlink implementation of the `netclass` collector.
```